website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -361,6 +361,150 @@ insert into analytics.replace_where_incremental
 </TabItem>
 </Tabs>
 
+<VersionBlock firstVersion="1.7">
+
+## Selecting compute per model
+
+Beginning in version 1.7.2, you can assign which compute to use on a per-model basis.
```

Comment:
```suggestion
Beginning in version 1.7.2, you can assign which compute resource to use on a per-model basis. For SQL models, you can select a SQL Warehouse (serverless or provisioned) or an all purpose cluster. Job Clusters may only be used for python models. 
```

Reply:
No, job clusters will never support SQL.  At this point we're contemplating removing even their python support, as they have no test coverage, documentation, or internal appetite to support.  All purpose clusters and sql warehouse both work.

Reply:
> No, job clusters will never support SQL. At this point we're contemplating removing even their python support, as they have no test coverage, documentation, or internal appetite to support. All purpose clusters and sql warehouse both work.

I guess I shouldn't say never, so much as I should say, because job clusters do not have a Thrift server, there is no path for them to work with SQL today.